stable/polaris/Chart.yaml

```
@@ -1,7 +1,7 @@
 apiVersion: v1
 description: Validation of best practices in your Kubernetes clusters
 name: polaris
-version: 4.2.0
+version: 4.2.1
```

The chart `4.2.1` is already released.
```suggestion
version: 4.2.2
```